src/aliceVision/matchingImageCollection/ImagePairListIO.cpp

```diff
@@ -0,0 +1,135 @@
+// This file is part of the AliceVision project.
+// Copyright (c) 2022 AliceVision contributors.
+// Copyright (c) 2012 openMVG contributors.
+// This Source Code Form is subject to the terms of the Mozilla Public License,
+// v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "ImagePairListIO.hpp"
+#include <aliceVision/system/Logger.hpp>
+#include <boost/algorithm/string/split.hpp>
+#include <boost/algorithm/string/trim.hpp>
+#include <fstream>
+
+namespace aliceVision {
```

Comment:
The code for each module should be in a namespace with the same name.

Reply:
Fixed now, thanks!